docs/home/howto/CLUSTER.md

```diff
@@ -440,17 +462,17 @@ To setup the cluster do the following steps from the provisioning machine:
       default_os_image: default
     ```
 
-    The [region](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html) lets you chose the most optimal place to deploy your cluster. The `subscription_name` is the Azure subscribtion under which you want to deploy the cluster.
+    The [region](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.RegionsAndAvailabilityZones.html) lets you chose the most optimal place to deploy your cluster. The `subscription_name` is the Azure subscription under which you want to deploy the cluster.
```

Comment:
“Optimal,” already contains the concept of being the most suitable or desirable thing. So saying “most optimal” would be like saying “most most desirable" :)

Reply:
![image](https://user-images.githubusercontent.com/13080132/135868095-7399a464-cc30-49f4-8853-44a4031f553a.png)